### ISSUE TITLE
copilot job environment

### ISSUE THREAD

**Issue by carlcontinusys** [aws/copilot-cli#1736]
It looks like schedule is required when we have environments on the manifest.

We have to add
on:
    schedule
on the environments too.
example:
environments:
   test:
       on:
           schedule: "expersion"

if not added, it will generate an error schedule required.

**Comment by efekarakus** [comment]
Heya @carlcontinusys 👋 

Thanks for reaching out to us! 

Are you saying that when you try to write environment overrides (https://aws.github.io/copilot-cli/docs/manifest/scheduled-job/#environments) for the "test" environment you have to duplicate again the "on" section?

Is this the current behavior?
```yaml
on: 
   schedule: @daily 
environments:
  test:
     on: 
        schedule: @daily 
     cpu: 512
```
but what you want is just:
```yaml
on: 
   schedule: @daily 
environments:
  test:
     cpu: 512
```

Thanks!

**Comment by carlcontinusys** [comment]
Yes

**Comment by efekarakus** [comment]
Apologies for the delay on this! It completely dropped off my radar, we sent a fix for it in #2044

**Comment by camilosantana** [comment]
@efekarakus should this issue be closed since #2043 was closed with #2044 and #2043 is a duplicate of this issue?

*UPDATE* ignore! i was looking at a stale webUI. issue already closed. disregard!
